src/features/user/TreeMapper/Treemapper.d.ts

```diff
@@ -1,4 +1,4 @@
-declare namespace Treemapper {
+export declare namespace Treemapper {
```

Comment:
You do not need to export a namespace. Once declared it is accessible throughout the application.